app/Controllers/Ijazah.php

```diff
@@ -119,4 +119,114 @@ public function delete($id)
         }
         return json_encode(["message" => "Invalid Method"]);
     }
-}
+
+    public function list()
+    {
+        // Jika user belum login redirect ke halaman login
+        // Jika fungsi login pada aplikasi ini sudah berjalan uncomment kode di bawah
+        // if (empty($this->session->get('user_id'))) return redirect("login");
+
+        // Sanitize request input saat mengakses URL
+        $vars = $this->request->getGet(null, FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+
+        // Nilai Deafult Variable
+        $defDraw = 1;
+        $defStart = 0;
+        $defLength = 10;
+        $defOrder = 'id';
+        $defSort = 'desc';
+        $defSearch = null;
+
+        // Deklarasikan variable draw    
+        if (!empty($this->request->getVar('draw'))) {
```

Comment:
kamu sudah simpan variabel $vars di baris 130 kenapa ga dipakai? saya ng memorinya simpan variabel tapi ga dipakai. malah call ulang `!empty($this->request->getVar('draw'))`
tentukan mana yang mau dipakai? juga di atas kamu pakai sanitasi kok yg ini ga?